ui/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -188,7 +208,7 @@ export default class PermissionConnect extends Component {
         ) : null}
         {isRequestingAccounts ? (
```

Comment:
Why is this hidden if no accounts are requested? :thinking: This results in the page number being hidden for the snaps install flow as well.

Reply:
The "back" button should be shown on the snap install screen as well, which would require the `goBack` function being updated to handle either page 2 or 3.

Reply:
The back button is a harder thing to implement - it would require removing already approved permissions, due to time constraints I suggest we leave it without a back button.

Reply:
I hadn't realized that the snap installation was handled by a separate permission request. I agree that postponing the pagination makes sense for now, including the back functionality.